htdocs/commande/card.php

```diff
@@ -2919,6 +2917,8 @@
 			// Note that $action and $object may be modified by hook
 			$reshook = $hookmanager->executeHooks('addMoreActionsButtons', $parameters, $object, $action);
 			if (empty($reshook)) {
+				$numlines = !empty($numlines) ? $numlines : count($object->lines);
```

Comment:
$numlines seems never defined before. So why not just 
$numlines = count($object->lines);

Reply:
I assume that it can be defined in a hook, am I wrong ?

Reply:
You're right. I made that choice before deleting the previous $numlines initialisation.